WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -17,10 +17,14 @@ protocol PointOfSaleAggregateModelProtocol {
     var cardReaderConnectionStatus: CardPresentPaymentReaderConnectionStatus { get }
     func connectCardReader()
     func disconnectCardReader()
+    var paymentState: PointOfSalePaymentState { get }
+    var cardPresentPaymentAlertViewModel: PointOfSaleCardPresentPaymentAlertType? { get set }
+    var cardPresentPaymentInlineMessage: PointOfSaleCardPresentPaymentMessageType? { get }
 
     @available(*, deprecated, message: "`allItems` is due for removal, use `itemListState` instead.")
     var allItems: [POSItem] { get }
     var itemListState: ItemListState { get }
+    var blockReturnToItemSelection: Bool { get }
```

Comment:
super nit: would a different verb tense make it more clear that it's a boolean and not a function?

```suggestion
    var blocksReturnToItemSelection: Bool { get }
```

Reply:
Yeah, agreed. In the options where I move it out of the aggregate model, I went with `shouldPreventCartEditing` as a descriptive boolean name – how does that work for you?